### ISSUE TITLE
[Question] Can recover_environment set render_mode attribute?

### ISSUE THREAD

**Issue by HanYuanHsu** [Farama-Foundation/Minari#179]
I tried to set the render mode in `recover_environment ` as is allowed in `gym.make`:

```
data_complete = minari.load_dataset("kitchen-complete-v1")
env = data_complete.recover_environment(render_mode="rgb_array")

```

But I got the following error:
```
TypeError: MinariDataset.recover_environment() got an unexpected keyword argument 'render_mode'
```

**Comment by younik** [comment]
Hey @HanYuanHsu,

This will be solved in the next release (hopefully soon); meanwhile you can install Minari from source to do this!

**Comment by younik** [comment]
We just released 0.4.3, and this should be fixed now
